src/client/config/helpers.js

```diff
@@ -0,0 +1,10 @@
+const openInNewTab = (url) => {
+  console.log('url: ', url);
+  const newWindow = window.open(url, '_blank', 'noopener,noreferrer');
+  if (newWindow) newWindow.opener = null;
```

Comment:
Brackets around `if` statements, please. Can we force it with `eslint` / `prettier`?